student/tests.py

```diff
@@ -1,15 +1,119 @@
 import pytest
+from django.contrib.auth.models import User
+from student.models import Student, EducationalInstitution
+from datetime import date
 
 
-def test_basic_math_operation():
-    assert 10 == 2 * 5
+@pytest.fixture
+def create_user():
+    user = User.objects.create_user("test", "testpassword")
+    user.save()
+    return user
 
 
 @pytest.fixture
-def numbers_list():
-    return [1, 2, 3, 4]
+def create_institution():
+    institution = EducationalInstitution(name="A")
+    institution.save()
+    return institution
+
+
+@pytest.fixture
+def create_student(create_user, create_institution):
```

Comment:
Fixtures are things, therefore fixture function names should typically be nouns and not verbs. Consider some of the code you have below:

`create_student.user.id`

It would make more since if it was like the following right?

`example_student.user.id`

Reply:
Got it.